tests/test_project_management.py

```diff
@@ -0,0 +1,222 @@
+        assert captor[0].url == TestCreateAndroidApp._CREATION_URL
+        body = {
+            'packageName': TEST_ANDROID_APP_PACKAGE_NAME,
+            'displayName': TEST_ANDROID_APP_DISPLAY_NAME,
+        }
+        assert json.loads(captor[0].body.decode()) == body
+        assert captor[1].method == 'GET'
+        assert captor[1].url == TestCreateAndroidApp._POLLING_URL
+        assert not captor[1].body
+        assert captor[2].method == 'GET'
+        assert captor[2].url == TestCreateAndroidApp._POLLING_URL
+        assert not captor[2].body
+
+    def test_create_android_app_already_exists(self):
+        captor = self._set_up_mock_responses_and_request_captor_for_project_management_service(
+            statuses=[409], responses=[ERROR_RESPONSE])
+
+        with pytest.raises(project_management.ApiCallError) as excinfo:
+            project_management.create_android_app(
+                package_name=TEST_ANDROID_APP_PACKAGE_NAME,
+                display_name=TEST_ANDROID_APP_DISPLAY_NAME)
+
+        assert 'The resource already exists' in str(excinfo.value)
+        assert excinfo.value.detail is not None
+        assert len(captor) == 1
+
+    def test_create_android_app_polling_rpc_error(self):
+        captor = self._set_up_mock_responses_and_request_captor_for_project_management_service(
+            statuses=[200, 200, 503],  # Error 503 means that backend servers are over capacity.
+            responses=[
+                OPERATION_IN_PROGRESS_RESPONSE,  # Request to create Android app asynchronously.
+                OPERATION_IN_PROGRESS_RESPONSE,  # Creation Operation is still not done.
+                ERROR_RESPONSE,  # Error 503.
+            ])
+
+        with pytest.raises(project_management.ApiCallError) as excinfo:
+            project_management.create_android_app(
+                package_name=TEST_ANDROID_APP_PACKAGE_NAME,
+                display_name=TEST_ANDROID_APP_DISPLAY_NAME)
+
+        assert 'Backend servers are over capacity' in str(excinfo.value)
+        assert excinfo.value.detail is not None
+        assert len(captor) == 3
+
+    def test_create_android_app_polling_failure(self):
+        captor = self._set_up_mock_responses_and_request_captor_for_project_management_service(
+            statuses=[200, 200, 200],
+            responses=[
+                OPERATION_IN_PROGRESS_RESPONSE,  # Request to create Android app asynchronously.
+                OPERATION_IN_PROGRESS_RESPONSE,  # Creation Operation is still not done.
+                OPERATION_FAILED_RESPONSE,  # Operation is finished, but terminated with an error.
+            ])
+
+        with pytest.raises(project_management.ApiCallError) as excinfo:
+            project_management.create_android_app(
+                package_name=TEST_ANDROID_APP_PACKAGE_NAME,
+                display_name=TEST_ANDROID_APP_DISPLAY_NAME)
+
+        assert 'Polling finished, but the Operation terminated in an error' in str(excinfo.value)
```

Comment:
Should be "operation" (lowercase) -- Fix where the error is raised.

Reply:
Done.